lib/AppInfo/Application.php

```diff
@@ -45,8 +45,13 @@ public function register() {
 
 		Util::connectHook('OC_Filesystem', 'preSetup', $this, 'addStorageWrapper');
 
-		$request = $this->getContainer()->getServer()->getRequest();
-		if (!\OC::$CLI && strpos($request->getPathInfo(), '/login') !== 0
+		// Only display the app on index.php except for public shares
```

Comment:
? Wasn't the actual idea to also show this to users in public shares? Because then the TOS are already applicable. Or did I get this app wrong?

Reply:
No, this app only works for logged in users

Reply:
Ah - okay 👍